### ISSUE TITLE
Colmap build does not recognise CUDA installation (Linux Mint 19.2 / CUDA 11.2)

### ISSUE THREAD

**Issue by lidgaca** [colmap/colmap#1117]
I've been through the build instructions for colmap and cannot get a build that recognises my CUDA installation. Thus I cannot access dense reconstructions.

System:
Linux Mint 19.2 (Ubuntu 18.04 bionic) All updates applied
Nvidia Graphics driver 460.32.03
Cuda 11.2
CMake 3.10.2

nvidia-smi shows
Sat Feb 13 11:28:23 2021

NVIDIA-SMI 460.32.03    Driver Version: 460.32.03    CUDA Version: 11.2
0  GeForce RTX 2060    Off  | 00000000:01:00.0  On |                  N/A |

**To Reproduce**

git clone ...
cd colmap
git checkout dev
mkdir build
cd build
cmake ..

(snippet of output ...)

-- Enabling SIMD support
-- Enabling OpenMP support
-- Disabling interprocedural optimization
-- Disabling CUDA support
-- Enabling OpenGL support
-- Disabling profiling support

**Expected behavior**
I expect to see that CUDA is enabled ...
I know that CUDA is basically working as I also use meshroom on this system, and I successfully used it today 

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Linux Mint 19.2 / Ubuntu 18.04 bionic

**Additional context**
I have cmake traces and have looked at them I can forward them if required, but they are rather large ... 
Unfortunately I'm not familiar with cmake :( When looking for the CUDA stuff in CMakeLists.txt line 112 +

set(CUDA_MIN_VERSION "7.0")
if(CUDA_ENABLED)
    find_package(CUDA ${CUDA_MIN_VERSION} QUIET)
endif()

the find_pakage() runs /usr/share/cmake-3.10/Modules/FindCUDA.cmake. This routine DOES set CUDA_FOUND = TRUE, but later in CMakeLists.txt line 192 CUDA_FOUND is tested and found to be false.

**Comment by lidgaca** [comment]
Additional Info:

I have also tried

    CC=/usr/bin/gcc-6 CXX=/usr/bin/g++-6 cmake ..

    and just using git clone (ie missing the 'git checkout dev' step) 

without success. I get the same results.

**Comment by lidgaca** [comment]
Apologies.

This has been resolved. During an automated update a few days ago Ubuntu (or Mint ?) decided that my CUDA installation needed updating. As I had loaded CUDA manually, this knocked out the nvcc compiler, leaving the runtime intact. This caused my confusion as I had assumed that CUDA was present and working.

A bit of reprogramming with an axe has cured the problem (until the next upgrade cycle ? :) )
